bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/DefaultSystemChannelTypeProvider.java

```diff
@@ -188,21 +195,24 @@
             .withDescription("Change the sound volume of a device")
             .withStateDescription(
                     new StateDescription(BigDecimal.ZERO, new BigDecimal(100), null, "%d %%", false, null))
-            .withCategory("SoundVolume").build();
+            .withCategory("SoundVolume")
+            .withTags(Arrays.asList(SystemTags.PROP_NOISE, SystemTags.CAP_DIMMABLE, SystemTags.CAP_VOLUME)).build();
 
     /**
      * Mute: default system wide {@link ChannelType} which allows muting and un-muting audio
      */
     public static final ChannelType SYSTEM_MUTE = ChannelTypeBuilder
             .state(new ChannelTypeUID(BINDING_ID, "mute"), "Mute", "Switch").withDescription("Mute audio of the device")
-            .withCategory("SoundVolume").build();
+            .withCategory("SoundVolume").withTags(Arrays.asList(SystemTags.PROP_NOISE, SystemTags.CAP_SWITCHABLE))
+            .build();
 
     /**
      * Media-control: system wide {@link ChannelType} which controls a media player
      */
     public static final ChannelType SYSTEM_MEDIA_CONTROL = ChannelTypeBuilder
             .state(new ChannelTypeUID(BINDING_ID, "media-control"), "Media Control", "Player")
-            .withCategory("MediaControl").build();
+            .withCategory("MediaControl").withTags(Arrays.asList(SystemTags.PROP_NOISE, SystemTags.CAP_CONTROL))
+            .build();
```

Comment:
I would remove "Property:Noise". Maybe we need a "Property:Music" for this and the others above. wdyt?

Reply:
Maybe "Property:Media"? because its not only about music, but also about video for example...

Reply:
go ahead and add it both here and to the wiki page :)